www/src/styles/theme.ts

```diff
@@ -11,7 +12,7 @@ declare module '@material-ui/core/styles/createMuiTheme' {
 const palette: PaletteOptions = {
     primary: {
         main: '#79B178',
-        light: '#E2F8E2',
+        light: '#ABD3AB',
```

Comment:
Why is this changing?

Reply:
This duplicated the background color so I thought this isn't what we want. I want to revisit this theme config in the future so we can also match the volunteer's table colors and adjust the typography font weights